src/interrogate/coverage.py

```diff
@@ -35,13 +35,16 @@ class BaseInterrogateResult:
     def perc_covered(self):
         """Percentage of node covered.
 
+        .. versionchanged:: 1.3.1
```

Comment:
Thanks so much for adding this - this helps a lot 😄  This should actually be `1.3.2` since `1.3.1` is already released - but I need to make the release now so I'll merge this in and then fix it in another commit.